Gemfile.lock

```diff
@@ -104,6 +104,7 @@ GEM
 PLATFORMS
   ruby
   universal-darwin-19
+  universal-darwin-20
```

Comment:
I don't think we are supposed to change this file in the repo? Or am I misunderstanding?

Reply:
I kind of thought I'd just sneak this into this PR hehe. 

@yingzhuchin 
This file updated after I was prompted yesterday to run `bundle install` on my machine before I could do a local build. IIUC, Big Sur runs on Darwin 20 (and I'm running Big Sur), while Catalina runs on Darwin 19, so I assume this is necessary for the build to work on Big Sur? IDK though. Just conjecture.

Reply:
I'm on Big Sur as well and `bundle install` doesn't add new platforms to my Gemfile.lock. What version of bundler are you running?

Reply:
Hmm...

Bundler version 2.2.1

I'll just remove the change from this PR so I can merge.